### PR TITLE
fix(http.js): add epsagon-trace-id extract

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: src/runners/express.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ const {
  */
 function createRunner(req, startTime) {
     const expressEvent = new event.Event([
-        `express-${uuid4()}`,
+        extractEpsagonHeader(req.headers) || `express-${uuid4()}`,
```
Comment (galbash) [inline src/runners/express.js]: maybe use it as another field in the metadata? will be more explicit and easier to port later

Comment (galbash) [reply src/runners/express.js]: (everywhere)

Comment (ranrib) [reply src/runners/express.js]: 👍

File: src/http.js [hunk 2]
```diff
@@ -19,5 +19,16 @@ function getIgnoredEndpoints() {
...
 }
 
+
+/**
+ * Gets the Epsagon header if exists, otherwise undefined
+ * @param {Object} headers object
+ * @returns {String} Epsagon header value
+ */
+function extractEpsagonHeader(headers) {
+    return headers['epsagon-trace-id'];
```
Comment (galbash) [inline src/http.js]: are we sure headers can't be undefined?

Comment (ranrib) [reply src/http.js]: I don't think so, but I added a check

**Pull request merged**